packages/eslint-plugin-pf-codemods/src/rules/v6/tokensWarn/tokens-warn.ts

```diff
@@ -0,0 +1,3706 @@
+import { Rule } from "eslint";
+import {
+  getDefaultDeclarationString,
+  getDefaultImportsFromPackage,
+  getFromPackage,
+} from "../../helpers";
+import { ImportDeclaration, ImportSpecifier } from "estree-jsx";
+
+module.exports = {
+  meta: {},
+  create: function (context: Rule.RuleContext) {
+    const tokensPackage = "@patternfly/react-tokens";
+
+    const { imports: tokenSpecifiers } = getFromPackage(context, tokensPackage);
+
+    const defaultTokensWithDeclaration = getDefaultImportsFromPackage(
```

Comment:
Think this might be something worth helperizing?

Reply:
I don't know if we ever use it again, I feel like it is quite specific. Do you have some potential future use case on your mind?

Reply:
To clarify I don't mean for tokens explicitly, but something that returns the default imports file name and the import declaration in a nice easy to use format like this.

I feel like I've had a few places where it would've been nice so far, though I haven't dug through to find what those instances were. If you don't think it would likely be helpful it's always something we can revisit in the future if it turns out the other way.

Reply:
Yeah, you are right, I also ran to something similar maybe on the component-groups. I lean towards creating the helper as a followup, if we run to a problem where it would be helpful